scripts/docs

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
```

Comment:
This file seems useless considered we don't have any docs for the lib.

Reply:
I've removed this script